process/dataValidators/nilHeaderHandlerProcessValidator.go

```diff
@@ -0,0 +1,17 @@
+package dataValidators
+
+import "github.com/ElrondNetwork/elrond-go/data"
+
+// nilHeaderHandlerProcessValidator represents a header handler validator that doesn't check the validity of provided headerHandler
+type nilHeaderHandlerProcessValidator struct {
```

Comment:
too long name: suggestion HeaderValidator

Reply:
Would keep process for now as the interceptors will be refactored as a whole. Maybe will need a HeaderHandlerDataValidator. About handler part, yes, it can be removed.

Reply:
Renmed to HeaderValidator